dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -417,6 +449,24 @@ def test_merge_correlation(self, *mocks):
         np.testing.assert_array_almost_equal(expected_corr_mat,
                                       merged_profile.correlation_matrix)
 
+        # Merge between existing data and empty data that still has samples
+        data = pd.DataFrame({'a': [1, 2, 4, np.nan, np.nan, np.nan],
+                             'b': [5, 7, 1, np.nan, np.nan, np.nan]})
+        data1 = data[:3]
+        data2 = data[3:]
+
+        profile1 = dp.StructuredProfiler(data1, options=profile_options)
+        expected_corr_mat = np.array([
+            [1, -0.78571429],
+            [-0.78571429,  1]
+        ])
+        np.testing.assert_array_almost_equal(expected_corr_mat,
+                                             profile1.correlation_matrix)
+        profile2 = dp.StructuredProfiler(data2, options=profile_options)
+        merged_profile = profile1 + profile2
+        np.testing.assert_array_almost_equal(expected_corr_mat,
+                                             merged_profile.correlation_matrix)
```

Comment:
should it be None in this case?

Reply:
I'm not sure. The second profile has `total_samples > 0`, but no actual values. Should the correlation matrix exist for data in this case? What do you think? @AnhTruong

Reply:
I think this is correct. It isn't that the values are preventing the calculation, it's that no data can be used to calculate.